boards/nxp/frdm_ke17z/frdm_ke17z.dts

```diff
@@ -74,3 +74,31 @@
 &gpioe {
 	status = "okay";
 };
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x0 DT_SIZE_K(44)>;
+		};
+		/* The MCUBoot swap-move algorithm uses the last 2 sectors
+		 * of the primary slot0 for swap status and move.
+		 */
+		slot0_partition: partition@b000 {
+			label = "image-0";
+			reg = <0xb000 (DT_SIZE_K(96) + DT_SIZE_K(8))>;
```

Comment:
4kB extra should suffice. Distribute the 4 kB over slot0 and slot1.

Reply:
Updated